Index: .idea/webContexts.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"WebContextManager\">\r\n    <option name=\"state\">\r\n      <map>\r\n        <entry key=\"file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views/magazineDetail.jsp\" value=\"file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views\" />\r\n        <entry key=\"file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views/memberLogin2.jsp\" value=\"file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views\" />\r\n      </map>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/webContexts.xml b/.idea/webContexts.xml
--- a/.idea/webContexts.xml	(revision 3a85328adc7fc936e7a6b046b270333303b4e125)
+++ b/.idea/webContexts.xml	(date 1653617662846)
@@ -4,6 +4,7 @@
     <option name="state">
       <map>
         <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views/magazineDetail.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views/memberJoin.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views" />
         <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views/memberLogin2.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/views" />
       </map>
     </option>
