Index: .idea/sqldialects.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"SqlDialectMappings\">\r\n    <file url=\"file://$PROJECT_DIR$/../../../../ojdbc8.jar\" dialect=\"GenericSQL\" />\r\n    <file url=\"PROJECT\" dialect=\"Oracle\" />\r\n  </component>\r\n</project>
===================================================================
diff --git a/.idea/sqldialects.xml b/.idea/sqldialects.xml
--- a/.idea/sqldialects.xml	
+++ b/.idea/sqldialects.xml	
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="SqlDialectMappings">
-    <file url="file://$PROJECT_DIR$/../../../../ojdbc8.jar" dialect="GenericSQL" />
     <file url="PROJECT" dialect="Oracle" />
   </component>
 </project>
\ No newline at end of file
